src/ontology/hp-edit.owl

```diff
@@ -27029,9 +27186,14 @@ AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/HP_0000717> "Th
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/HP_0000717> "Autism")
 SubClassOf(<http://purl.obolibrary.org/obo/HP_0000717> <http://purl.obolibrary.org/obo/HP_0000729>)
 
-# Class: <http://purl.obolibrary.org/obo/HP_0000718> (Aggressive behavior)
+# Class: <http://purl.obolibrary.org/obo/HP_0000718> (Aggressive behaviour)
```

Comment:
UK spelling?

Reply:
This should be "behavior". It looks like the synonyms include both spellings.